spouse/tf_model.py

```diff
@@ -2,56 +2,85 @@
 from typing import Tuple
 import numpy as np
 import pandas as pd
+from snorkel.types import DataPoint
+
 import tensorflow as tf
+from tensorflow.keras.layers import (
+    Bidirectional,
+    Concatenate,
+    Dense,
+    Embedding,
+    Input,
+    LSTM,
+)
+
+
```

Comment:
Is there any way to reuse the preprocessors that get the left, right and between tokens? 

https://github.com/snorkel-team/snorkel-tutorials/blob/b6f85b89e623e2dd00ca23bf2572390fd7dd0b80/spouse/preprocessors.py#L55

Reply:
And is person1 always to the left of person2? If so, then the min() shouldn’t be needed